src/main/java/com/synopsys/integration/detect/lifecycle/run/step/IntelligentModeStepRunner.java

```diff
@@ -131,9 +131,16 @@ public void runOnline(
         });
 
         stepHelper.runToolIfIncluded(DetectTool.CONTAINER_SCAN, "Container Scanner", () -> {
```

Comment:
Will Container Scan run if detect.tools is not set explicitly with values other than CONTAINER_SCAN?

Reply:
You mean if detect.tools is not set with CONTAINER_SCAN but is set with other values? In that case, container scan won't run. 
If we specify anything in `detect.tools`, only those tools are run. 

If there are a list of tools mentioned in `detect.tools` and if we also specify `detect.tools.excluded`, then out of the `detect.tools` provided, the excluded tools won't run.

But, by default, if `detect.tools` is not set at all, all tools get included, except the ones provided in `detect.tools.excluded` (if that is provided)